_posts/2020-08-28-acess.md-the-most-open-privacy-first-emr-solution.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Access.MD – the most open, privacy first, EMR solution"
```

Comment:
Should be Primary Care EMR , since its a huge distinction from specialist EMRS

Reply:
```suggestion
title: "ACCESS.MD – the most open, privacy first,  primary care EMR solution"
```

so something like this ?